ipamutils/utils.go

```diff
@@ -37,18 +37,30 @@ type NetworkToSplit struct {
 // InitNetworks initializes the broad network pool and the granular network pool
 func InitNetworks(defaultAddressPool []*NetworkToSplit) {
 	initNetworksOnce.Do(func() {
-		// error ingnored should never fail
-		PredefinedGranularNetworks, _ = splitNetworks(defaultGranularNetwork)
+		// error ignored should never fail
+		PredefinedGlobalScopeDefaultNetworks, _ = splitNetworks(globalScopeDefaultNetworks)
 		if defaultAddressPool == nil {
-			defaultAddressPool = defaultBroadNetwork
+			defaultAddressPool = localScopeDefaultNetworks
 		}
 		var err error
-		if PredefinedBroadNetworks, err = splitNetworks(defaultAddressPool); err != nil {
-			logrus.WithError(err).Error("InitAddressPools failed to initialize the default address pool")
+		if PredefinedLocalScopeDefaultNetworks, err = splitNetworks(defaultAddressPool); err != nil {
+			logrus.WithError(err).Error("InitAddressPools failed to initialize the local scope default address pool")
 		}
 	})
 }
 
+// ConfigGlobalScopeDefaultNetworks initializes the global scope default network pool
+func ConfigGlobalScopeDefaultNetworks(defaultAddressPool []*NetworkToSplit) {
```

Comment:
if you define this function as: `func ConfigGlobalScopeDefaultNetworks(defaultAddressPool *NetworkToSplit...) {`
You can avoid from other places to call it with nil, but you will be able to call the method as:
`ConfigGlobalScopeDefaultNetworks()` that is nicer

Reply:
swarm init is the only place we are going to call. we can reuse the same function for swarm update. We are not going to call from any other place.

Reply:
sorry I was referring to the fact that there is a nil check at line 54. If that is needed, means that this method can be called with nil, I was just suggesting that in that case, would be nicer to have a method that takes 0 arguments instead of 1 argument set to nil

Reply:
Added unit test to cover new changes